src/uwtools/set_namelist.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
This file should not be included as a part of this PR as it is no longer a part of the repository and has been moved to `templater.py`.